SmartSaver.MVC/Views/Authentication/Login.cshtml

```diff
@@ -9,29 +9,57 @@
 </head>
 
 <br />
-<h1>User Login</h1>
 <br />
+<body style="background-size:cover; background-image: url(https://images.unsplash.com/photo-1475650522725-015d35677789?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=1050&q=80)">
+<section class="login-block">
+    <div class="card center" style="max-width: 500px; padding: 20px 20px 20px 20px;">
+        <div class="card-block">
+            <div class="col-md-12">
+                <form method="post" asp-action="Login" asp-controller="Authentication" asp-route-returnUrl="@Context.Request.Query["returnurl"]">
+
+                    <p class="h4 text-center py-4">User Login</p>
+
+                    <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+                    <div class="form-group form-primary">
+                        <input asp-for="Email" class="form-control" placeholder="Email">
+                        <span asp-validation-for="Email" class="text-danger"></span>
+                    </div>
+                    <div class="form-group form-primary">
+                        <input asp-for="Password" type="password" class="form-control" placeholder="Password" />
+                        <span asp-validation-for="Password" class="text-danger"></span>
+                    </div>
+                    <button type="submit" class="btn btn-primary btn-md btn-block waves-effect text-center m-b-20" style="padding:5px 35px 5px 35px;">Login</button>
+                </form>
+                <div class="or-container">
+                    <div class="line-separator"></div>
+                    <div class="or-label">or</div>
+                    <div class="line-separator"></div>
+                </div>
+                <form method="post" asp-action="ExternalLogin" asp-controller="Authentication" asp-route-returnUrl="@Context.Request.Query["returnurl"]">
+                    <div class="row">
+                        <div class="col-md-12">
+                            <button type="submit" name="provider" value="Google" class="btn btn-lg btn-google btn-block text-uppercase btn-outline">
+                                Google Login⠀
+                                <img src="https://img.icons8.com/color/16/000000/google-logo.png">
+                            </button>
+                        </div> <br>
```

Comment:
<br /> element is unnecessary here, it does not do anything. However, if you are trying to achieve more space between elements with <br />, you should better use mb-1 , mb-2, ... , mt-1, mt-2 ... bootstrap classes.